docs/_static/auth/v1.yaml

```diff
@@ -191,6 +191,7 @@ paths:
 
         error returns:
         * 400 - one of the parameters has the wrong format
+        * 502 - the authorizer could not be reached or returned an error
```

Comment:
This should be part of the generic errors (502) which might suggest that 502 is a bad idea to return when the remote server isn't responding. 503 might be better. 502's are returned by reverse proxies when the internal server is offline. Not really part of this PR, may be a new issue.